plugins/parodos/src/components/workflow/useGetProjectAssessmentSchema.ts

```diff
@@ -68,31 +68,30 @@ export function useGetProjectAssessmentSchema({
 
   const formSchema = jsonSchemaFromWorkflowDefinition(cloned);
 
-  set(formSchema, `steps[0].uiSchema.onboardingAssessmentTask.['ui:order']`, [
-    'newProject',
-    'name',
-    'description',
-    '*',
-  ]);
+  set(
+    formSchema,
+    `steps[0].uiSchema.[${ASSESSMENT_WORKFLOW_TASK}].['ui:order']`,
```

Comment:
I think I would just do it right now rather than a quick fudge.

Why is has it got `${}` around the constant?

I can create the PR if it is unclear how.

Reply:
I did not see it was a string literal, ignore that

Reply:
I can create the PR that uses the app-config if you want.

Reply:
yes please Thank you